src/LanguageServer.ts

```diff
@@ -915,24 +937,171 @@ export class LanguageServer {
         this.connection.sendNotification('build-status', 'success');
     }
 
-    private async onDefinition(params: TextDocumentPositionParams): Promise<Location[]> {
-        //WARNING: This only works for a few small xml cases because the vscode-brightscript-language extension
-        //already implemented this feature, and I haven't had time to port all of that functionality over to
-        //this codebase
-        //TODO implement for brs/bs also
+    public async onWorkspaceSymbol(params: WorkspaceSymbolParams) {
+        const results = util.flatMap(
```

Comment:
We probably need `await this.waitAllProgramFirstRuns();` at the start of this function since there's no guarantee that all the workspaces are loaded when it gets activated.